src/components/Banner/BannerButton.tsx

```diff
@@ -0,0 +1,27 @@
+import "./BannerButton.scss";
+import { FiChevronRight } from "react-icons/fi";
+import cx from "classnames";
+
+type Props = {
+  label: string;
```

Comment:
let's use `children` instead?

Reply:
I didn't use children because we need so the sparkle next to the last word. So I just want to make sure the label is a string. Children can also be a ReactNode or Element